engine/src/main/scala/cromwell/engine/callactor/CallActor.scala

```diff
@@ -25,11 +25,23 @@ object CallActor {
   sealed trait CallActorMessage
   sealed trait StartMode extends CallActorMessage {
     def executionMessage: CallExecutionActorMessage
+    def maybeCallLogs: Option[CallLogs]
   }
-  case object Start extends StartMode { override val executionMessage = CallExecutionActor.Execute }
-  final case class Resume(jobKey: JobKey) extends StartMode { override val executionMessage = CallExecutionActor.Resume(jobKey) }
-  final case class UseCachedCall(cachedBackendCall: BackendCallJobDescriptor, backendCall: BackendCallJobDescriptor) extends StartMode {
+  case object StartFinalCall extends StartMode {
+    override val executionMessage = CallExecutionActor.Execute
+    override val maybeCallLogs = None
+  }
+  final case class StartBackendCall(maybeCallLogs: Option[CallLogs]) extends StartMode {
+    override val executionMessage = CallExecutionActor.Execute
+  }
+  final case class Resume(jobKey: JobKey) extends StartMode {
+    override val executionMessage = CallExecutionActor.Resume(jobKey)
+    override val maybeCallLogs = None
```

Comment:
Am I correct to assume that the use case for these `maybeCallLogs` is exclusively for messaging back to the `WorkflowActor`, so we don't need or want to pass any existing value for these logs in with a `Resume` object?